src/analysis/retail/warlock/demonology/modules/talents/DreadCalling.tsx

```diff
@@ -0,0 +1,60 @@
+import { formatPercentage } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import { TALENTS_WARLOCK } from 'common/TALENTS';
+import { TooltipElement } from 'interface';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { CastEvent } from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+
+/**
+ * Each Soul Shard spent on Hand of Gul'dan increases the damage of your next Call Dreadstalkers by 4%.
+ */
+
+class DreadCalling extends Analyzer {
+  callDreadstalkersCasts: number = 0;
+  dreadCallingStacksConsumed: number = 0;
+
+  DAMAGE_BUFF_PER_STACK = 0.04;
```

Comment:
Define this as a const variable outside of the class since its not really needed as a member variable